web-marketplace/src/legacy-pages/Orders/hooks/useOrders.tsx

```diff
@@ -29,22 +29,34 @@ import { ORDER_STATUS } from 'components/organisms/Order/Order.constants';
 
 import { Order } from '../Orders.types';
 
-export const useOrders = () => {
+type UseOrdersParams = {
+  address?: string | null;
+};
+
+export const useOrders = ({ address }: UseOrdersParams = {}) => {
   const apolloClient = useApolloClient() as ApolloClient<NormalizedCacheObject>;
   const { queryClient } = useLedger();
   const { activeAccount, loading, privActiveAccount } = useAuth();
   const { activeWalletAddr } = useWallet();
   const [selectedLanguage] = useAtom(selectedLanguageAtom);
 
-  const fiatOrders = useMemo(
-    () =>
+  const buyerAddress = address ?? activeWalletAddr;
+
+  const fiatOrders = useMemo(() => {
```

Comment:
ditto we probably do not need to change this file either
BTW this would still show fiat orders for the current account, not the org
we haven't implemented any back end support for that since it's not in the spec